common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -38,7 +38,7 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-contributions-epic-articles-viewed-month",
+    "ab-contributions-epic-articles-viewed-round-3",
     "States how many articles a user has viewed in the epic in a month",
```

Comment:
Does this sentence need updating to reflect that the period is flexible?

Reply:
👍 good spot